### ISSUE TITLE
virtio-fs doesn't aware modified unless refresh

### ISSUE THREAD

**Issue by fecet** [virtio-win/kvm-guest-drivers-windows#945]
**Describe the bug**

After modify file system on host, windows guest have to refresh to know taht.

**To Reproduce**
Steps to reproduce the behaviour:

Write something on host

**Expected behavior**

Changes happen on guest as soon as host

**Screenshots**
![image](https://github.com/virtio-win/kvm-guest-drivers-windows/assets/41792945/14849491-f880-4a53-9a40-1e207e956dcf)

**Host:**
 - Disto: Ubuntu
 - Kernel version: 5.0.4
 - QEMU version 7.0
 - QEMU command line start by virt-manager
 - libvirt version 8.6
 - libvirt XML file

**VM:**
 - Windows 10
 - virtiofs
 - 1.229

**Additional context**

**Comment by viktor-prutyanov** [comment]
Hi @fecet,

Windows Explorer shows a state as of the most recent ReadDirectory return. But any new request to FS will work with an updated state.

From an architectural point of view, Explorer can show an updated state with ReadDirectoryChangesW, but in the virtiofs service it requires a processing of the notify VirtIO queue. Also I'm not sure that WinFsp has full support of such FS to kernel notify logic.

To sum up, this is hypothetically feasible, but requires a lot of investigation and development. I'm not sure this is a good investment of time for just seeing host updates in Explorer.

**Comment by fecet** [comment]
of course we dont care Explorer but for some reason I need to modify file on host and watch the changes in guest. Is watching file soulds like a interesting feature good to have?

My current solution is share smb from guest, it's really slow

**Comment by viktor-prutyanov** [comment]
All changes are seen on the guest side

**Comment by viktor-prutyanov** [comment]
Explorer simply doesn't show them

**Comment by fecet** [comment]
So it's perhaps beeter be a FR, add watching file support for guest

**Comment by viktor-prutyanov** [comment]
OK, so let's reopen it and mark it as a feature request.

**Comment by xiagao** [comment]
jira issue: https://issues.redhat.com/browse/RHEL-12348

**Comment by xiagao** [comment]
@YanVugenfirer I have no privilege to add label, could you update with 'Jira_Attached' label.

**Comment by YanVugenfirer** [comment]
This issue has been tracked in Jira: https://redhat.atlassian.net/browse/RHELMISC-34289
